client/js/widgets/widget.js

```diff
@@ -2315,8 +2315,9 @@ export class Widget extends StateManaged {
       if(widgetType != 'card' && widgetType != 'pile')
         continue;
 
-      // check if this widget is closer than 10px from another widget in the same parent
-      if(widget.get('parent') == thisParent && Math.abs(widget.get('x')-thisX) < 10 && Math.abs(widget.get('y')-thisY) < 10) {
+      // check if this widget is closer than the pileSnapRange from another widget in the same parent
+      let pileSnapRange = (thisType == 'card' ? this.get('onPileCreation').pileSnapRange : this.get('pileSnapRange')) || defaultPileSnapRange;
```

Comment:
`this.get('onPileCreation').pileSnapRange` is not _quite_ save. The property `onPileCreation` could be `null`. For example if inherited from another widget type.